core/src/main/scala/filodb.core/reprojector/MapDBMemTable.scala

```diff
@@ -84,6 +84,8 @@ class MapDBMemTable[K](val projection: RichProjection[K], config: Config) extend
 
   def numRows: Int = rowMap.size().toInt
 
+  def forceCommit(): Unit = {}
+
   def clearAllData(): Unit = {
     logger.info(s"MemTable: ERASING ALL TABLES!!")
```

Comment:
Seeing phrases in tools like Splunk with double exclamation points paired with words like 'erasing' can be frightening. Just sayin'.